Agilefreaks/FreaksCatalog-API
# [130] Add photo to freaks

## Description
Create a Photo Model and a table in database for photos.
For each freak will be saved in database an url to his profile picture, which will pe uploaded on [Gravatar](https://ro.gravatar.com/site/implement)

## Related

[[130] Add Photo to a Freak](https://trello.com/c/0yKfo1ER/130-add-photo-to-a-freak)

## Steps to Test or Reproduce
You can run a test from here `spec/controllers/graphql/mutations/photo_create_spec.rb`.
Also, you can run the test with Insomnia or by executing a GraphQl mutation from localhost:

1. Start rails server by typing `rails s `command in terminal
2. Access [localhost/graphiql](http://127.0.0.1:3000/graphiql) or use Insomnia to execute a GraphQl mutation
3. By executing this mutation you can see that everything works fine
```GraphQl
mutation {
    photoCreate(uri: "https://www.gravatar.com/avatar/00000000000000000000000000000000") {
        uri
    }
}
`
